### DISCUSSION TITLE
Customizing the User listing page?

### DISCUSSION THREAD

**Discussion by ARCHES-Academy** [statamic/cms#4372] [Q&A]
In v2, I know that you could add a `.yaml` file to the base user directory (described at [this](https://v2.statamic.com/control-panel#customizing-listing-columns) link (https://v2.statamic.com/control-panel#customizing-listing-columns). Is that still possible in v3? The default columns are irrelevant for my use case, I want to change them. I can't find any ways to customize them, however, and the `folder.yaml` doesn't seem to change anything. 

Can the default columns be changed for the user listing?

**Comment by duncanmcclean** [comment]
No, it's not currently possible. There is an open feature request for this however, statamic/ideas#146.

**Reply by ARCHES-Academy** [reply]
That's truly unfortunate. I hope that it gets implemented soon. Until then, we're going to have to build a custom cp view to list users in a data-list table.
